docs/api/classifier/threshold.rst

```diff
@@ -32,19 +32,19 @@ A visualization of precision, recall, f1 score, and "queue rate" with respect to
 
 .. image:: images/spam_discrimination_threshold.png
 
-One common use of binary classification algorithms is to use the score or probability they produce to determine cases that require special treatment. For example, a fraud prevention application might use a classification algorithm to determine if a transaction is likely fraudulent and needs to be investigated in detail. In the figure above, we present an example where a binary classifier determines if an email is "spam" (the positive case) or "not spam" (the negative case); emails that are detected as spam are moved to a hidden folder and eventually deleted.
+One common use of binary classification algorithms is to use the score or probability they produce to determine cases that require special treatment. For example, a fraud prevention application might use a classification algorithm to determine if a transaction is likely fraudulent and needs to be investigated in detail. In the figure above, we present an example where a binary classifier determines if an email is "spam" (the positive case) or "not spam" (the negative case). Emails that are detected as spam are moved to a hidden folder and eventually deleted.
 
-Many classifiers use either a ``decision_function`` to score the positive class or a ``predict_proba`` function to compute the probability of the positive class. If the score or probability is greater than some *discrimination threshold* then the positive class is selected, otherwise the negative class is.
+Many classifiers use either a ``decision_function`` to score the positive class or a ``predict_proba`` function to compute the probability of the positive class. If the score or probability is greater than some discrimination threshold then the positive class is selected, otherwise, the negative class is.
 
-Generally speaking, the threshold is balanced between cases and set to 0.5 or 50% probability. However, this threshold may not be the optimal threshold: often there is an inverse relationship between *precision* and *recall* with respect to a discrimination threshold. By adjusting the threshold of the classifier, it is possible to tune the F1 score (the harmonic mean of precision and recall) to the best possible fit or to adjust the classifier to behave optimally for the specific application. Classifiers are tuned by considering the following metrics:
+Generally speaking, the threshold is balanced between cases and set to 0.5 or 50% probability. However, this threshold may not be the optimal threshold: often there is an inverse relationship between precision and recall with respect to a discrimination threshold. By adjusting the threshold of the classifier, it is possible to tune the F1 score (the harmonic mean of precision and recall) to the best possible fit or to adjust the classifier to behave optimally for the specific application. Classifiers are tuned by considering the following metrics:
```

Comment:
Removed the inline formatting to precision and recall since they do not need to be italicized.